dvc/env.py

```diff
@@ -9,3 +9,4 @@
 DVC_IGNORE_ISATTY = "DVC_IGNORE_ISATTY"
 DVC_EXP_GIT_REMOTE = "DVC_EXP_GIT_REMOTE"
 DVC_EXP_AUTO_PUSH = "DVC_EXP_AUTO_PUSH"
+DVC_ANALYTICS = "DVC_ANALYTICS"
```

Comment:
This should be called `DVC_NO_ANALYTICS` as it's sole purpose is to disable analytics.

See also: https://keepsake.ai/docs/learn/analytics

Reply:
@skshetry It is also meant to force-enable analytics, so `NO_ANALYTICS` would get confusing.

Reply:
Why does the user need to force-enable it? Feels unnecessary.

Reply:
For the record: continuing in https://github.com/iterative/dvc.org/pull/2724